crowdsourcing/viewsets/project.py

```diff
@@ -89,9 +89,9 @@ class ModuleViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         queryset = Module.objects.all()
-        requesterid=self.request.query_params.get('requesterid',None)
```

Comment:
but, won't it return the requester in  Response ? , I want requester object to filter out the modules ! 
Plus, I was thinking of keeping this viewset more general, like, it can also be used to show portfolio of a worker also, by sending request /api/modle/?workerportfolio=3 or something like that !